packages/noya-state/src/selectors.ts

```diff
@@ -152,6 +174,16 @@ export const getSelectedColorSwatches = (
   );
 };
 
+export const getSelectedLayerStyle = (
```

Comment:
If this returns an array, should the name be plural? I.e. `getSelectedLayerStyles`